nuget/Lykke.СryptoFacilities/Models/Common/OrderType.cs

```diff
@@ -0,0 +1,20 @@
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Lykke.СryptoFacilities.Models.Common
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum OrderType
```

Comment:
Add default value to None (unspecified) to avoid errors while requesting with empty order type.